docs/mp/guides/36_graalnative.adoc

```diff
@@ -1,6 +1,6 @@
 ///////////////////////////////////////////////////////////////////////////////
 
-    Copyright (c) 2020 Oracle and/or its affiliates.
+    Copyright (c) 2019, 2021 Oracle and/or its affiliates.
```

Comment:
Was it intentional to change the original copyright year from 2020 to 2019? Maybe it should become "2020, 2021" instead?